static/js/form-helpers.js

```diff
@@ -5,7 +5,7 @@ window.onload = (event) => {
     const request_account_label = document.querySelector('label[for="id_why_account_needed"]');
     const department_label = document.querySelector('label[for="id_department"]');
 
-    const domainRegex = /^(?:[A-Z0-9](?:[\.A-Z0-9-]{0,61}[A-Z0-9])?\.)*?(uu|lu|gu|su|umu|liu|ki|kth|chalmers|ltu|hhs|slu|kau|lnu|oru|miun|mau|mdu|bth|fhs|gih|hb|du|hig|hh|hkr|his|hv|ju|sh)\.se$/i;
+    const domainRegex = /^(?:(?!\b(?:student|stud)\b\.)[A-Z0-9](?:[\.A-Z0-9-]{0,61}[A-Z0-9])?\.)*?(uu|lu|gu|su|umu|liu|ki|kth|chalmers|ltu|hhs|slu|kau|lnu|oru|miun|mau|mdu|bth|fhs|gih|hb|du|hig|hh|hkr|his|hv|ju|sh)\.se$/i;
```

Comment:
https://regex101.com/ shows me that this regex doesn't recognize something like `student.uu.se`

Reply:
<img width="739" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/49a095fa-db6f-498f-90c9-1b5fc4fd3003">

blah.uu.se doesn't work for me with this regexp :/

Reply:
I think you are doing a case sensitive match but it needs to be insensitive (click on "gm" to add that option)
![Screenshot 2024-01-25 at 17 19 12](https://github.com/ScilifelabDataCentre/stackn/assets/16312524/a963668f-6c8a-4a6f-a83e-20e875518999)